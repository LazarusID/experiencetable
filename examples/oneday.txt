Ron:      1
Harry:    2
Hermione: 3