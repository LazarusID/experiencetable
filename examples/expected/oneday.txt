Harry:    2
Hermione: 3
Ron:      1
